### ISSUE TITLE
Feed not showing up

### ISSUE THREAD

**Issue by dominik-sfl** [bluesky-social/feed-generator#34]
I am not entirely sure how this works, but I'd be interested in getting it working. If I wanted to publish a very simple feed generator that filters for a certain phrase / hashtag, what would I need to do exactly? 

I just followed the quick start and ran publishFeed a couple of times, but it's not showing up on bluesky.

I am also wondering whether I don't need to change the uri, which is not mentioned in the quickstart, as well as how to find out my `did`?:

```
export const uri = 'at://did:example:alice/app.bsky.feed.generator/whats-alf'
```

**Comment by dholms** [comment]
you can get your did with
`https://bsky.social/xrpc/com.atproto.identity.resolveHandle?handle=${yourHandle}`

this unfortunately won't work running locally, you'll need to deploy your feed somewhere

**Comment by dominik-sfl** [comment]
Thanks for your reply. So I need to deploy the feed to some nodejs server (e.g. EC2 or Heroku) and then run `publishFeed` on that instance? 

And then here, I'll also have to replace the did from the url you provided in here, like so?:

```
// /src/alogs/whats-alf.ts
export const uri = 'at://did:${MY_DID_GOES_HERE}/app.bsky.feed.generator/whats-alf'
```

**Comment by NetWalker108** [comment]
> Edit: Does this also mean that I need to register a domain name for the feed generator? In the .env file it asks for a domain name. I'm a bit confused as to how I can get going with this as fast and lightweight as possible.

Yeah you'll have to get a hostname(your server provider would usually provide you with one automatically) so Hostname will be "did:web:{YOUR_REMOTE_SERVER_PUBLIC_DOMAIN}"

**Comment by dholms** [comment]
Yup as @NetWalker108 said, you'll need a domain name

You have the uri formatting correct 👍

**Comment by chukaibejih** [comment]
@NetWalker108 @dholms 

Hey guys I could use some help here. I followed the README and edited only the files mentioned. 

- I ran yarn published in my terminal and it was successful then I deployed the repo on the railway
- for the HOSTNAME, it was the railway URL (eg `some-app.railway.com`) and the feed URI is `export const URI = 'at://did:web:some-app.railway.com/app.bsky.feed.generator/feed-name`
- After deployment, it says the app cant be reached.
- Idk if I am messing up a variable or the port 

![image](https://github.com/bluesky-social/feed-generator/assets/29266211/c2a0f1af-e4c0-420e-bc52-b066c1d3368f)

**Comment by dominik-sfl** [comment]
Any success or news so far? I will also try this on railway tonight or tomorrow afternoon.

**Comment by chukaibejih** [comment]
> Any success or news so far? I will also try this on railway tonight or tomorrow afternoon.

To be fair, I switched to the Python SDK and used render instead od railway. I successfully deployed and I can see the feed on the Blusky app but It is an empty list even though the three endpoints on my domain return the appropriate files.  
```
- /.well-known/did.json
 - /xrpc/app.bsky.feed.describeFeedGenerator
 - /xrpc/app.bsky.feed.getFeedSkeleton
```
I am not so sure what the error could be 
![image](https://github.com/bluesky-social/feed-generator/assets/29266211/3a2d42c1-591a-4a01-b953-f5be72f6cc5d)
![image](https://github.com/bluesky-social/feed-generator/assets/29266211/3597a7c1-6952-4b99-ad6b-064cfc249064)

**Comment by andrewfleer** [comment]
Did you have to do anything special to deploy on Render? 

I've been planning to use Netlify, and I can get the python app "deployed" but all the endpoints return a 404 error.

I then tried on Render, but it wouldn't build because of an invalid Flask version. Ideally I'd like to run this on Netlify, as I have more experience there but I don't know why this isn't working for me.

**Comment by chukaibejih** [comment]
can i see a screenshot of the error?

**Comment by andrewfleer** [comment]
Eh, I'm not too worried about it anymore. I used [skyfeed.app](https://skyfeed.app) to build my feed instead

**Comment by Baldemar-Loza** [comment]
@chukaibejih Did you end up figuring this out? Im running into the exact same issue you were having.
